test/modules/fundingManager/bondingCurve/utils/mocks/FM_BC_Bancor_Redeeming_VirtualSupplyV1Mock.sol

```diff
@@ -104,6 +107,22 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Mock is
             _calculateNetAndSplitFees(_totalAmount, _protocolFee, _workflowFee);
     }
 
+    function call_handleIssuanceTokenAfterBuy(address _receiver, uint _amount)
```

Comment:
Should be `exposed_` right? But leave it if the test of the mock is still old-style.